src/stores/ConfigStore.ts

```diff
@@ -2,26 +2,20 @@ import { createStore } from './utils';
 
 import type { AccessModel, Config } from '#types/Config';
 import type { AdSchedule } from '#types/ad-schedule';
+import { ACCESS_MODEL, INTEGRATION } from '#src/config';
 
-export enum PersonalShelf {
-  ContinueWatching = 'continue_watching',
-  Favorites = 'favorites',
-}
-
-export const PersonalShelves = [PersonalShelf.Favorites, PersonalShelf.ContinueWatching];
-
-type CleengData = {
-  cleengId: string | null | undefined;
-  cleengSandbox: boolean;
-  monthlyOfferId: string;
-  yearlyOfferId: string;
+export type IntegrationInfo = {
+  integrationType: keyof typeof INTEGRATION | null;
+  useSandbox: boolean;
+  clientId?: string;
+  offers: string[];
 };
 
 type ConfigState = {
   config: Config;
   accessModel: AccessModel;
   adScheduleData: AdSchedule | null | undefined;
-  getCleengData: () => CleengData;
+  getIntegration: () => IntegrationInfo;
```

Comment:
Do we need this to be method or could we add integrationType, sandbox, clientId, and offers to the store as properties and just set them once when the config is set?

Reply:
Well, we can do it, but these props seem to be config's derivatives and fully depend just on the config value to be calculated. So I thought it could make sense to make them part of the store.

Reply:
Right, I agree. What I was suggesting was to just do the calculation derivation once and then set the values instead of having a method that recalculates each time.